delivery/repositories/project_repository.py

```diff
@@ -123,116 +136,184 @@ def dir_contains_fastq_files(d):
-            os.path.join(report_dir, "report.xsl")
+        # look for the MultiQC report in these locations (in order of precedence)
+        report_paths = [
+            project_path,
+            os.path.join(
+                runfolder.path,
+                "seqreports",
+                "projects",
+                project_name
+            )
         ]
-        report_files.extend(list(
-            self.filesystem_service.list_files_recursively(
-                os.path.join(
-                    report_dir,
-                    "Plots"))))
-        return report_files
-
-    @staticmethod
-    def multiqc_report_path(project):
-        return os.path.join(
-            project.path,
-            "{}_multiqc_report.html".format(project.name))
-
-    def multiqc_report_files(self, project):
-        report_files = [self.multiqc_report_path(project)]
-        report_dir = self.filesystem_service.dirname(report_files[0])
-        report_files.append(
-            os.path.join(report_dir, "{}_multiqc_report_data.zip".format(project.name)))
-        return report_files
+        # depending on the location of the MultiQC report, use these different prefixes
+        report_prefixes = [
+            project_name,
+            f"{runfolder.name}_{project_name}"
+        ]
+        report_names = [
+            "multiqc_report.html",
+            "multiqc_report_data.zip"
+        ]
+        for report_path, report_prefix in zip(report_paths, report_prefixes):
+            # creating the RunfolderFile for a non-existing file should raise an exception
+            try:
+                report_files = []
+                for report_name in report_names:
+                    report_file = os.path.join(
+                        report_path,
+                        f"{report_prefix}_{report_name}"
+                    )
+                    report_files.append(
+                        RunfolderFile.create_object_from_path(
+                            file_path=report_file,
+                            runfolder_path=runfolder.path,
+                            filesystem_service=self.filesystem_service,
+                            metadata_service=self.metadata_service,
+                            base_path=report_path,
+                            checksums=checksums
+                        )
+                    )
+            except FileNotFoundError:
+                report_files = []
+            # if no exception was raised, return the report files and log a message depending on
+            # where they were found
+            else:
+                if report_path == report_paths[0]:
+                    log.info(
+                        f"MultiQC reports found in Unaligned/{project_name}, overriding "
+                        f"organisation of seqreports"
+                    )
+                else:
+                    log.info(f"Organising seqreports for {project_name}")
+
+                return report_files
```

Comment:
```suggestion
                finally:
                    return report_files
```
Shouldn't this be part of a `finally:` block? I mean if a `FileNotFoundError` is raised you still need to return `report_files` right?

Reply:
It will raise an exception if no `report_files` could be returned

Reply:
Aha I see, I guess it doesn't really matter then, I think I see what you meant now when you said you're not sure if it's an improvement, at least the function is self contained now